chart/compass/charts/director/config.yaml

```diff
@@ -101,4 +130,6 @@ clientCredentialsRegistrationScopes:
     - "label_definition:read"
     - "label_definition:write"
     - "automatic_scenario_assignment:read"
-    - "automatic_scenario_assignment:write"
\ No newline at end of file
+    - "automatic_scenario_assignment:write"
+    - "integration_system.auths:read"
```

Comment:
one of the recommendations in the Treat modeling workshop is that we don't return credentials that were provided to us to the entity that provided them. So maybe we could remove line 134.